src/components/SignInPage/SignInPage.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import ErrorMessage from './Error/ErrorMessage';
+import {SignInPageProps, SignInPageState} from './index';
+import InputField from './Input/InputField';
+
+const StyledSignInForm = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  color: #000;
+  transition: none;
+`;
+
+const StyledForm = styled.div`
+  width: 300px;
+  border: 1px solid #ccc;
+  padding: 40px;
+`;
+
+const StyledButtonWrap = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`;
+
+class SignInPage extends React.Component<SignInPageProps, SignInPageState> {
+  constructor(props: SignInPageProps) {
+    super(props);
+    this.state = {
+      commonError: '',
+      email: '',
+      emailError: '',
+      password: '',
+      passwordError: ''
+    };
+  }
+
+  componentDidMount() {
+    document.body.addEventListener('keyup', this.keyupHandler);
+  }
+
+  componentWillUnmount() {
+    document.body.removeEventListener('keyup', this.keyupHandler);
+  }
+
+  keyupHandler = (e: any) => {
```

Comment:
i believe handling Enter keypress can be done just by wrapping in the `form` element